src/markdown-pages/install-with-kurl/reducing-storage.md

```diff
@@ -0,0 +1,211 @@
+---
+path: "/docs/install-with-kurl/changing-storage"
+date: "2022-10-10"
+weight: 27
+linktitle: "Changing Storage"
+title: "Changing Storage"
+isAlpha: false
+---
+
+It is relatively common to initially allocate less (or more) storage than is required for an installation in practice.
+Some storage providers allow this to be done easily, while others require far more effort.
+
+
+# OpenEBS LocalPV
+
+OpenEBS LocalPV consumes a host's storage at `/var/openebs/local`.
+If you increase the amount of storage available here, OpenEBS will use it.
+If it shrinks, OpenEBS will have less storage to use.
+
+# Rook-Ceph
+
+## Expanding Storage
+
+Rook does not support expanding the storage of existing block devices, only adding new ones.
+However, a 100GB block device added to a node that already had a 100GB block device used by rook will be treated similarly to a freshly created instance with a single 200GB disk.
+
+## Contracting Storage
+
+Rook does not support shrinking a block device once it has been allocated to Rook.
+However, entire block devices can be removed from the cluster.
```

Comment:
Should we make a note here, or maybe it's done below, that you need to keep the minimum of 3 nodes with sufficient storage for your current usage?